### ISSUE TITLE
Dropdown not working

### ISSUE THREAD

**Issue by michelboaventura** [kaliber5/ember-bootstrap#338]
@srvance asked me to open this issue with a reproducible case of the drop-down component not working. It is located at: https://github.com/michelboaventura/eb_dropdown

This repo has only two commits. The first one is just the ember installation and the second one has everything I've changed to use ember-bootstrap. Since I've put the drop-down on application.hbs, you should see the problem by just running ember and accessing the root route. 

Thanks!

---

Output from `ember bootstrap:info` (available since 1.0.0-alpha.9):
```
Npm packages:
ember-bootstrap: ^1.0.0-alpha.12 -> 1.0.0-alpha.12
ember-cli: 2.13.2 -> 2.13.2
bootstrap: 4.0.0-alpha.5 -> 4.0.0-alpha.5
bootstrap-sass: n/a
ember-cli-sass: n/a
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapFont: true
importBootstrapCSS: true
```

**Comment by srvance** [comment]
@michelboaventura To make sure the configuration is correct, you should use the generator to switch to BS4.

```
ember g ember-bootstrap --bootstrap-version=4
```

In this case, it would have installed `bootstrap@next` which resolves to "4.0.0-alpha.6" which works just fine.

As a side note, it doesn't affect the operation here, but it also would have set `importBootstrapFont` to `false` because BS4 doesn't include glyphicons anymore.

**Comment by michelboaventura** [comment]
Hu @srvance, so it was a problem with bootstrap ``4.0.0-alpha.5`` and no ember-bootstrap. Thank you very much for taking the time!

Cheers
